internal/core/operations/add_rooms/add_rooms_executor_test.go

```diff
@@ -93,7 +93,7 @@ func TestAddRoomsExecutor_Execute(t *testing.T) {
 		roomsManager := mockports.NewMockRoomManager(mockCtrl)
 
 		schedulerStorage.EXPECT().GetScheduler(context.Background(), op.SchedulerName).Return(&scheduler, nil)
-		roomsManager.EXPECT().CreateRoomAndWaitForReadiness(gomock.Any(), gomock.Any()).Return(&gameRoom, &gameRoomInstance, nil).Times(10)
+		roomsManager.EXPECT().CreateRoomAndWaitForReadiness(gomock.Any(), gomock.Any(), gomock.Any()).Return(&gameRoom, &gameRoomInstance, nil).Times(10)
```

Comment:
@caiordjesus i think we can use the false value in this mock expectation since we don't expect any add room to be a validation one, right?